package.json

```diff
@@ -218,7 +215,7 @@
   },
   "engines": {
     "node": "8.10.0",
-    "yarn": "1.5.1"
+    "yarn": "1.12.3"
```

Comment:
There was an `integrity` property in our `yarn.lock` that was overwritten when I ran `yarn install` with `1.5.1`, so I did some searching to find that we must have checked-in a  `yarn.lock` version generated by a new version of Yarn, without upgrading this property as well.